contracts/Baal.sol

```diff
@@ -431,9 +431,9 @@ contract Baal is Module {
             abi.encode(
                 DOMAIN_TYPEHASH,
                 keccak256(bytes(name)),
+                keccak256(bytes("4")),
```

Comment:
just an fyi - version 1 is really annoying, v4 format is much more fluid with the 712 standard